extensions/wikia/PageHeaderExperiments/js/localSearchSuggestions.js

```diff
@@ -0,0 +1,78 @@
+require(['mw', 'jquery', 'wikia.window'], function(mw, $, window) {
+	mw.loader.using('jquery.autocomplete').then(function () {
+
+		var $searchInput = $('#searchInputTest');
+		var $searchInputWrapper = $('#searchInputWrapperTest');
+		var searchSuggestionsUrl = $searchInput.data('suggestions-url');
+		var autocompleteReEscape = new RegExp('(\\' + ['/', '.', '*', '+', '?', '|', '(', ')',
+				'[', ']', '{', '}', '\\'].join('|\\') + ')', 'g');
+
```

Comment:
What do you think about checking if this experiment is active (experiment id is in `window.optimizely.activeExperiments`)? I will also check is `$searchInput` defined and if not, send tracking data with such information.